conf/zeppelin-site.xml.template

```diff
@@ -485,6 +485,14 @@
 </property>
 -->
 
+<!--
+<property>
+    <name>zeppelin.server.send.jetty.name</name>
+    <value>true</value>
```

Comment:
I would use `false` here, as it's `true` by default anyway, and people will uncomment this section only when they need to disable sending of Jetty name ...

Reply:
I understand your "secure-by-design" rationale here. Expect a PR soon for several other configs.